examples/MinikubeDemo.md

```diff
@@ -144,7 +144,7 @@ Let's use [this](./workerConfigMap.yaml) template.
 ```
 kubectl apply -f workerConfigMap.yaml
 ```
-This template will share among blow three demos(Grid, Hyperband, and GPU).
+This template will share among below three demos(Grid, Hyperband, and GPU).
```

Comment:
"This template will be shared by the three demos below (Grid, Hyperband, and GPU).

Reply:
thanks!
Updated